frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1990,15 +2035,24 @@ mod tests {
 			assert_eq!(
 				multi_phase_events(),
 				vec![
-					Event::UnsignedPhaseStarted { round: 1 },
+					Event::PhaseTransition {
```

Comment:
FYI, in case in some tests you don't want to match on all events, you can use:

```
matches!(&some_vec[..], [first, ..., something, before, last]))
```

etc.